### PR TITLE
Add juno "lucina" testnet

### PR DESCRIPTION
**Pull request by dimiandre**
Add juno "lucina" testnet to registry

### CODE REVIEW

**Review by ethanfrey — APPROVED** [review]
Looks good.
Please add a changelog (patch) as described here: https://github.com/confio/ts-relayer/blob/main/DEVELOPMENT.md#create-a-changeset

Then I am happy to merge this (just got back from vacation)

**Comment by ethanfrey** [comment]
@dimiandre can you try adding a changelog please? happy to merge

File: .changeset/eight-eggs-own.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@confio/relayer': patch
+---
+
+Added Juno testnet details to registry
```
Comment (ethanfrey) [inline .changeset/eight-eggs-own.md:5]: thanks for adding this

**Review by ethanfrey — APPROVED** [review]
lgtm

**Comment by dimiandre** [comment]
> @dimiandre can you try adding a changelog please? happy to merge

Done! Sorry for the delay.

File: .changeset/eight-eggs-own.md (hunk 1 above)
Comment (dimiandre) [reply .changeset/eight-eggs-own.md:5]: Thanks you! sorry for the delay!

**Pull request merged**